test/e2e/tests/swap-send/swap-send-eth-with-loaded-state.spec.ts

```diff
@@ -0,0 +1,297 @@
+import { Suite } from 'mocha';
+import { MockttpServer } from 'mockttp';
+import {
+  logInWithBalanceValidation,
+  openActionMenuAndStartSendFlow,
+  withFixtures,
+} from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import SettingsPage from '../../page-objects/pages/settings/settings-page';
+import AdvancedSettings from '../../page-objects/pages/settings/advanced-settings';
+import HomePage from '../../page-objects/pages/home/homepage';
+import { NATIVE_TOKEN_SYMBOL, SwapSendPage } from './swap-send-test-utils';
+
+const RECIPIENT_ADDRESS = '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1';
```

Comment:
nit: if we want to use a variable, we could use the existing one `DEFAULT_FIXTURE_ACCOUNT` from the `constants.ts` file